assets/src/edit-story/components/canvas/carousel/index.js

```diff
@@ -95,6 +97,9 @@ const PlainStyled = styled(Plain)`
 `;
 
 const MenuIconsWrapper = styled.div`
+  & > * {
+    margin-bottom: 12px;
+  }
```

Comment:
Totally cool with a quick fix here and don't need to change, but would like to actively try to keep specificity down when coming up with future solutions.

Wanna avoid tracking down which parent up the component tree is applying some styles as much as possible when visual regressions eventually come.

Reply:
Moved the margin-bottom the `SafeModeButton` styled component